backend/openapi.yaml

```diff
@@ -3298,13 +3302,225 @@ paths:
                 $ref: '#/components/schemas/SmsMessageTransactional'
         '400':
           description: Bad Request
-        "401":
+        '401':
           description: Unauthorized
-        "429":
+        '429':
           description: Too many requests. Please try again later.
-        "500":
+        '500':
           description: Internal Server Error
+  /transactional/sms:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - SMS
+      summary: 'List transactional SMSes'
+      parameters:
+        - in: query
+          name: limit
+          description: max number of messages returned
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 100
+            default: 10
+        - in: query
+          name: offset
+          description: offset to begin returning messages from
+          required: false
+          schema:
+            type: integer
+            minimum: 0
+            default: 0
+        - in: query
+          name: status
+          description: status of messages to filter for
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum: [UNSENT, ACCEPTED, SENT, DELIVERED, ERROR]
+          style: form
+          explode: true
+        - in: query
+          name: created_at
+          description: >
+            Filter for created_at timestamp of messages:
+              - gt: greater than
+              - gte: greater than or equal
+              - lt: less than
+              - lte: less than or equal
+          required: false
+          schema:
+            type: object
+            minProperties: 1
+            properties:
+              gt:
+                type: string
+                format: date-time (ISO 8601)
```

Comment:
I think the `date-time` field is inconsistent between `date-time` and `date-time (ISO 8601)`. think we can standardise on the latter if it's more precise? also can specify our time zone to be more precise?